py-polars/pyproject.toml

```diff
@@ -22,9 +22,12 @@ connectorx = ["connectorx"]
 profile = "black"
 
 [tool.mypy]
-ignore_missing_imports = true
 disallow_untyped_defs = true
 files = ["polars", "tests"]
 
+[[tool.mypy.overrides]]
```

Comment:
So the idea is that when these libs get stubs we can include them one by one?

Reply:
Yes, either stubs (such as Pandas) or add a `py.typed` file to denote the type annotations in the code can be used (like numpy, and polars itself)

Reply:
Oh. so libraries can have different tactics? We added a `py.typed`, but you can also create a separate `stubs` package?